.nix/devnet.nix

```
@@ -108,6 +106,23 @@ let
       };
     };
   };
+
+  dali-persistent-machine = mkPersistentMachine {
+    domain = "persistent.devnets.composablefinance.ninja";
+    machine-name = "composable-persistent-devnet";
+    ip = "persistent-devnet-ip";
+    package = "devnet-persistent";
+    chain = "dali";
+  };
+
+  picasso-persistent-machine = mkPersistentMachine {
+    domain = "persistent.picasso.devnets.composablefinance.ninja";
```

Thanks for your effort sir,

Is it possible to have a different URL scheme for these reflecting the function of them?
Eg:
```suggestion
    domain = "picasso-staging.composable.finance";
```

---

Let's open a ticket please because this require an extra roundtrip with SRE to setup the dns